src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -49,125 +55,146 @@ export class QuickBar extends LitElement {
 
   @internalProperty() private _activatedIndex = 0;
 
-  @query("paper-input", false) private _filterInputField?: HTMLElement;
+  @query("search-input", false) private _filterInputField?: HTMLElement;
 
-  @query("mwc-list-item", false) private _firstListItem?: HTMLElement;
+  @query("mwc-list", false) private _list?: HTMLElement;
 
   public async showDialog(params: QuickBarParams) {
     this._commandMode = params.commandMode || false;
-    this._opened = true;
     this._commandItems = this._generateCommandItems();
     this._entities = Object.keys(this.hass.states).map<HassEntity>(
       (entity_id) => this.hass.states[entity_id]
     );
+    this._opened = true;
   }
 
   public closeDialog() {
     this._opened = false;
-    this._itemFilter = "";
+    this._filter = "";
     this._commandTriggered = -1;
+    this._commandItems = undefined;
```

Comment:
Minor:

We could probably remove this line and let the command list stay cached for next launch since the command list will very rarely change in the middle of a session.